app/helpers/FileStorage/LocalStorage/LocalFileStorage.php

```diff
@@ -464,6 +467,39 @@ public function deleteOldFiles(string $glob, int $threshold): int
         return $deleted;
     }
 
+    public function deleteByFilter(callable $filter): int
+    {
+        $rootDirLen = strlen($this->rootDirectory) + 1; // plus 1 for '/';
```

Comment:
Is there really always `/` at the end? Or at the beginning? How can you know that if the path is not normalized or is given without `/`?

Reply:
+1 means '/' is always appended to the root directory and we want to remove the root dir prefix and the added '/'.

Reply:
I can't seem to find the place where it is ensured that the root always has '/' at the end...

Reply:
The RecursiveDirectorIterator appends '/' on Linux, '\' on Windows to the root dir.

Reply:
jesus, I see it now... I was looking at this from the wrong side... sorry